pkg/ui/workspaces/cluster-ui/src/indexDetailsPage/indexDetailsPage.tsx

```diff
@@ -379,11 +388,20 @@ export class IndexDetailsPage extends React.Component<
         />
       );
     }
+
+    const isCockroachCloud = this.context;
     // If no props are passed, render db-console breadcrumb links by default.
+    console.log(this.props.details.databaseID);
     return (
       <Breadcrumbs
         items={[
           { link: DB_PAGE_PATH, name: "Databases" },
+          {
+            link: isCockroachCloud
+              ? EncodeDatabaseUri(this.props.databaseName)
+              : databaseDetailsPagePath(this.props.details.databaseID),
```

Comment:
Why do we use the legacy database URI for CC here, but not for the table details page below?

Reply:
Oops that should also follow the same condiiton. Thanks for catching, done!